site/content/troubleshooting-guide.md

```diff
@@ -1 +1,122 @@
-### TODO
\ No newline at end of file
+
+# Troubleshooting Guide
```

Comment:
Add some tips on dealing with the bootstrap stack. In particular if the bucket stack can't be created due to the S3 bucket already existed. We say users have that problem. Also if the bootstrap stack is in a bad state how it might be best to just recreate it.